smallrye-reactive-messaging-camel/src/test/java/io/smallrye/reactive/messaging/camel/incoming/IncomingCamelTest.java

```diff
@@ -46,8 +46,12 @@ public void testWithABeanDeclaringACamelSubscriber() {
         initialize();
 
         await().until(() -> {
-            List<String> list = Files.readAllLines(path);
-            return list.size() == 1 && list.get(0).equalsIgnoreCase("abcd");
+            try {
```

Comment:
I'm not sure it's required. That was more a 'what the heck???' thing.

Reply:
I've removed it.